crates/forge_open_router/src/open_router.rs

```diff
@@ -167,32 +174,37 @@ impl ProviderService for OpenRouter {
     }
 
     async fn parameters(&self, model: &ModelId) -> Result<Parameters> {
-        // For Eg: https://openrouter.ai/api/v1/parameters/google/gemini-pro-1.5-exp
-        let path = format!("parameters/{}", model.as_str());
-
-        let url = self.url(&path)?;
-
-        let text = self
-            .client
-            .get(url)
-            .headers(self.headers())
-            .send()
-            .await?
-            .error_for_status()?
-            .text()
-            .await?;
-
-        let response: ParameterResponse = serde_json::from_str(&text)
-            .with_context(|| "Failed to parse parameter response".to_string())?;
-
-        Ok(Parameters {
-            tool_supported: response
-                .data
-                .supported_parameters
-                .iter()
-                .flat_map(|parameter| parameter.iter())
-                .any(|parameter| parameter == "tools"),
-        })
+        match self.provider {
+            ProviderKind::Ollama => Ok(Parameters::new(false)),
+            ProviderKind::OpenRouter => {
+                // For Eg: https://openrouter.ai/api/v1/parameters/google/gemini-pro-1.5-exp
```

Comment:
Let's drop ProviderKind and if OpenRouter is enabled (check base URL) then get this information.